### ISSUE TITLE
Set default value text

### ISSUE THREAD

**Issue by Alexendoo** [urfave/cli#504]
If I want to have a port flag that is random if not specified, it would be helpful to indicate that by having the default value display as `(default: random)` rather than 0 for example. Usage would be along the lines of

```go
cli.IntFlag{
    Name:        "port, p",
    DefaultText: "random",
},
```

An option to not display the default text would also work

**Comment by jszwedko** [comment]
Hi @Alexendoo,

This seems like a reasonable addition.

You could work around this deficiency for now by using a `cli.GenericFlag` where the `Value` used for has a `String()` method that returns `"random"`. Though you'd then also need to handle the int parsing yourself.

Thanks for the suggestion!

**Comment by jszwedko** [comment]
Fixed by #514 -- thanks again for suggesting this @Alexendoo
